src/main/java/com/google/growpod/controllers/UserDao.java

```diff
@@ -153,4 +154,35 @@ public boolean addToUserGardenList(String userId, String gardenId) {
     datastore.add(newEntity);
     return true;
   }
+
+  /**
+   * Deletes a garden id from the user's garden list.
+   *
+   * @param userId the user's id
+   * @param gardenId the garden's id
+   * @return whether the query was successful.
+   */
+  public boolean deleteFromUserGardenList(String userId, String gardenId) {
```

Comment:
nit: looks like you didn't carry over the new naming here.

Reply:
As I get things merged, I will carry forward the new naming.